scripts/cangen/cangen/traffic.py

```diff
@@ -0,0 +1,42 @@
+"""
+Author: Manush Patel
+Date: 2024-11-24
+https://macformula.github.io/racecar/firmware/can-traffic/
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass
+class CANMessage:
+    data_length: int
+    frequency: int
+
+
+def data_length(data_bytes):
+    fixed_length = 44  # fixed length of CAN frame
+    variable_length = data_bytes * 9.6  # accounted for bit stuffing
```

Comment:
this is a good comment since it is not immediately clear by we `*9.6` instead of `*8`

actually I just realized that bit stuffing should apply to the whole message, not just the data portion. therefore we should have `*8` here and `*1.2` on the total length